framework/src/controls/ControllableItem.C

```diff
@@ -11,7 +11,9 @@
 #include "ConsoleUtils.h"
 
 ControllableItem::ControllableItem(const MooseObjectParameterName & name,
-                                   libMesh::Parameters::Value * value)
+                                   libMesh::Parameters::Value * value,
+                                   std::set<ExecFlagType> flags)
```

Comment:
Should this be pass-by-const reference instead?

Reply:
Yes.